packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -265,12 +264,12 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
           localPosition: event.localPosition,
         );
       } else {
-        _pendingDragOffset += OffsetPair(local: event.localDelta, global: event.delta);
+        _pendingDragOffset = _pendingDragOffset + OffsetPair(local: event.localDelta, global: event.delta);
```

Comment:
Why's `+=` no good anymore?

Reply:
I should have made `_pendingDragOffset` nullable at some point and change it again later. Updated.